lib/omniauth/login_dot_gov/authorization.rb

```diff
@@ -28,12 +29,35 @@ def redirect_url_params
         }
       end
 
+      def aal
+        if client.aal.nil?
+          nil
+        elsif VALID_AAL_VALUES.include?(client.aal.to_s)
+          client.aal.to_s
+        else
+          raise "Invalid AAL, choose one of #{VALID_AAL_VALUES}"
+        end
+      end
+
       def acr_values
-        if client.ial == 2
+        values = []
+
+        values << if client.ial == 2
```

Comment:
looks like AAL is a string but this is an int, so a user of the gem might think this is a string too. Should we cast it to accept more consistent input?

```suggestion
        values << if client.ial.to_i == 2
```